drivers/at86rf215/at86rf215_netdev.c

```diff
@@ -782,6 +812,15 @@ static void _ack_timeout_cb(void* arg) {
     msg_send_int(&dev->ack_msg, dev->ack_msg.sender_pid);
 }
 
+static void _backoff_timeout_cb(void* arg) {
+    at86rf215_t *dev = arg;
+    if (!(dev->flags & AT86RF215_OPT_CCATX)) {
+        at86rf215_reg_write(dev, dev->RF->RG_EDC, 1);
+    } else {
+        at86rf215_rf_cmd(dev, CMD_RF_TXPREP);
+    }
+}
```

Comment:
This is called by the timer callback, so this is executed in an interrupt context.
Since the RIOT SPI API uses mutexes to protect the bus from concurrent transfers, you can't access SPI in an interrupt.

Reply:
So I need some thread do this? 
I think a new thread just to do CCA is to much over head.
Any idea which one I can use for this task?

Reply:
See 905ea3fd6829d9dcce707a3faed0517724466fbd
I had to do the same for the ACK timeout.

Reply:
ok.
which example do you use for that ping test

Reply:
Just `examples/gnrc_networking`